config/config.go

```diff
@@ -244,6 +244,22 @@ func (c *Configuration) BasicTransfersOnly() bool {
 	return basicOnly
 }
 
+// TusTransfersAllowed returns whether to only use "tus.io" HTTP transfers.
+// Default is false, including if the lfs.tustransfers is invalid
+func (c *Configuration) TusTransfersAllowed() bool {
+	value, ok := c.GitConfig("lfs.tustransfers")
```

Comment:
Looks like we have the [`GitConfigBool`](https://github.com/github/git-lfs/blob/c2ae16a30553ec38a21af5e174613f14c6d6fc0c/config/config.go#L424-L437) func which may simplify some of this 😄